roles/migrationcontroller/defaults/main.yml

```diff
@@ -44,3 +44,13 @@ velero_restic_restore_helper_image: "{{ registry }}/{{ project }}/{{ velero_rest
 velero_restic_restore_helper_repo: "{{ lookup( 'env', 'VELERO_RESTIC_RESTORE_HELPER_REPO') }}"
 velero_restic_restore_helper_version: "{{ snapshot_tag | default(lookup( 'env', 'VELERO_RESTIC_RESTORE_HELPER_TAG')) }}"
 velero_state: absent
+
```

Comment:
@pranavgaikwad @jwmatthews We're obviously dependent on having the NooBaa operator installed, but does it make sense to make NooBaa itself optional with a `noobaa: true` parameter or similar in the default.yaml and exposed and easily overrideable in the MigrationController CR so customers that don't want to use NooBaa can turn it off?